tests/Microsoft.Bot.Builder.TestBot.Json/Samples/GeneratedForm/luis/luis.settings.chrimc.westus.json

```diff
@@ -1,5 +0,0 @@
-{
-    "luis": {
-        "sandwich_en-us_lu": "236ff454-e019-4ab3-ad5f-0ae22d65ad33"
-    }
-}
```

Comment:
Why do you delete these?  What is our policy around these?  We need at least one checked-in for the mock stuff to work, but we also need people to be able to update the mock.  Whose settings win?